cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
 relationship: has_relation QC:4000013 ! QC metric relation: one run
 relationship: has_relation QC:4000067 ! Total ion current chromatogram
 
+
+[Term]
+id: QC:4000071
+name: Column type
+def: "The parent to the group of column definitions for name, content, and units for table type." [PSI:QC]
+relationship: has_relation id: QC:4000006 ! table
+
+[Term]
+id: QC:4000072
+name: Retention times
+def: "The column contains retention times in seconds." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: UO:0000010 ! second
+
+[Term]
+id: QC:4000073
+name: Relative intensities 
+def: "The column contains relative intensity values of one type." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: MS:1000043 ! intensity unit
+
+[Term]
+id: QC:4000074
+name: Sequence coverages
+def: "The column contains sequence coverage values in percent, e.g. from proteins." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: UO:0000187 ! percent
+
+[Term]
+id: QC:4000075
+name: Protein accessions
+def: "The column contains protein accessions." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C45253 ! String
+
+[Term]
+id: QC:4000076
+name: Length values
+def: "The column contains length values of sequences, i.e. the count of items within the sequences, e.g. total length of proteins." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000077
+name: Target or decoy designation
+def: "The column contains designations of the type 'target' or 'decoy'." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: NCIT:C45253 ! String
+
+[Term]
+id: QC:4000078
+name: Sample rates
+def: "The column contains sample rates as number of occurrences." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: PATO:0000161 ! rate
+
+[Term]
+id: QC:4000079
+name: Counts
+def: "The column contains counts of occurrences of a common type." [PSI:QC]
+is_a: QC:4000071 ! Column type
+has_units: STATO:0000047 ! count
+
+[Term]
+id: QC:4000080
+name: Peptide sequences
+def: "The column contains peptide sequences." [PSI:QC]
```

Comment:
can sequences carry modifications?

Reply:
I wouldn't want to restrict the contents too much. Would this likely break 'metric consumer code' if we'd say sequences in any common representation format?